test.sh

```diff
@@ -8,6 +8,25 @@
 
 set -e
 
+
+if [[ -f "./manifest.sh" ]]; then
+  echo "Files are generated by dist-gen. Check if everything generated by distgen properly."
+  echo "Generating all files by 'make generate-all' ..."
+  make generate-all
+  echo "Files are generated by distgen."
+  # common submodule is not updated at all. No argument to git diff is needed.
+  if ! git diff --ignore-submodules=all --exit-code; then
+    git show -s
+    echo "Pull request does not contain properly generated files."
+    echo "Please run these command in order to update pull request:"
```

Comment:
What about adding a information, that the distgen-generated files should be added in a separate commit?

Reply:
Make sense to me.